config/default/.htaccess

```diff
@@ -0,0 +1,24 @@
+# Deny all requests from Apache 2.4+.
```

Comment:
@jamesaldred This file doesn't seem to be related but rather something from local dev environment.

Reply:
@wouteradem this has now been corrected